### PR TITLE
update auth page

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review the PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,8 +37,8 @@
       </div>
 
       <nav class="navbar">
-        <a href="/" class="href"><i class="fas fa-home icons"></i>Home</a>
-        <a href="#" class="href"> <i class="fas fa-chart-line icons"></i>Dashboard</a>
+        <a href="/" class="href"><i class="fas fa-home social-quizicons"></i>Home</a>
+        <a href="#" class="href"> <i class="fas fa-chart-line social-quizicons"></i>Dashboard</a>
       </nav>
 
       <form action="" class="class-form">
```
Comment (Sharmaryan) [inline index.html:44]: action attribute can be removed

**Review by pratik-kumar-sahu — APPROVED** [review]
Looks good for merging go ahead

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          />
+          <a href="/" class="logo">Travizz Quiz</a>
+        </div>
+  
+        <nav class="navbar">
+          <a href="/" class="href"><i class="fas fa-home social-quizicons"></i>Home</a>
+          <a href="#" class="href"> <i class="fas fa-chart-line social-quizicons"></i>Dashboard</a>
+        </nav>
+  
+        <form action="" class="class-form">
```
Comment (pritam-kr) [inline pages/login.html:36]: action attribute can be removed But it is not an issue.

File: pages/question/question.html [hunk 3]
```diff
@@ -14,37 +14,40 @@
...
-      <h1 class="logo"><a href="/">Travizz</a></h1>
+    <header>
+      <div id="menu-bar" class="fas fa-bars menu-bar"></div>
+      <div class="logo-header">
+        <img
+          src=""
+          alt="logo"
+          class="logo-design"
+        />
+        <a href="/" class="logo">Travizz Quiz</a>
```
Comment (pritam-kr) [inline pages/question/question.html:28]: put logo URL if have.

**Review by Sharmaryan — APPROVED** [review]

File: pages/question/question2.html [hunk 4]
```diff
@@ -14,38 +14,41 @@
...
-    <header class="header">
-      <h1 class="logo"><a href="/">Travizz</a></h1>
+    <header>
+      <div id="menu-bar" class="fas fa-bars menu-bar"></div>
+      <div class="logo-header">
+        <img
+          src=""
+          alt="logo"
+          class="logo-design"
+        />
```
Comment (pritam-kr) [inline pages/question/question2.html:27]: it is good practice to add alt attributes with some text.

File: pages/stylesheets/auth.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
+.form-box {
+    width: 100%;
+    max-width: 60rem;
+    position: relative;
+    top: 10rem;
+    margin: 5rem auto;
+    /* background: var(--bg-code); */
```
Comment (pritam-kr) [inline pages/stylesheets/auth.css:7]: remove commented code

File: pages/stylesheets/quiz.css [hunk 6]
```diff
@@ -200,4 +198,4 @@ h3 {
 
 .result__ans {
   color: var(--blue-color);
-}
+}
```
Comment (pritam-kr) [inline pages/stylesheets/quiz.css:201]: break line.

**Review by pritam-kr — APPROVED** [review]
I have added some points but for suggestions only. 
Good to go. You can merge it.

**Comment by vivek-guda** [comment]
Good to merge. GOOD JOB!

**Pull request merged**